서영/단어뒤집기2.py

```diff
@@ -0,0 +1,31 @@
+# https://www.acmicpc.net/problem/17413
+
+from sys import stdin
+
+S = stdin.readline().rstrip()
+ans = ""
+
+
+def reverse_word(x, y):
+    global ans
+    words = S[x:y+1].split(" ")
```

Comment:
제가 파이썬 컨벤션에 대해 잘 몰라서 하는 말일수도 있는데요, 연산자 앞/뒤로 띄워주면 어떨까요?
제가 보기에는 조금 더 가독성이 좋아보여서요! (주관적인 의견입니다!)
```suggestion
    words = S[x:y + 1].split(" ")
```

Reply:
확실히 가독성이 좋아보이네요! 노력해봐야겠어요😄